pkg/controller/rest/verifiable/models.go

```diff
@@ -78,6 +80,12 @@ type getCredentialByNameReq struct { // nolint: unused,deadcode
 	Name string `json:"name"`
 }
 
+// PresentationRequest is model for verifiable credential.
+type PresentationRequest struct {
```

Comment:
Why can't you refer existing model from command?

Reply:
used existing model